css/style.css

```diff
@@ -0,0 +1,13 @@
+Html {
+    border-radius: 5px;
+    display: block;
+    margin-bottom: 22px;
+    max-width: 100%
+  }
+
+ body {
+    border-radius: 50%;
+    height: 130px;
+    margin: -66px 0 22px 0;
+    vertical-align: top;
+  }
```

Comment:
Nice job👍 your comment looks clear and polite, but let's make it even clearer by adding a reason for requesting the change😉 remember learnt in [the lesson on how to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) that we should "Remember to explain why you request a change - it might be obvious for you but not for someone else". kindly explain why you are requesting the change as you did in the first comment😉

Reply:
Good job aligning your project body with CSS stylesheet. Kindly fix the alignment and spacing of the elements of as well

Reply:
Good job aligning your project body with CSS stylesheet. Kindly fix the alignment and spacing of the elements of as well. Element selectors help you keep your code simple by applying the styling to all elements on a page of that type to keep track of your styles for those elements